highfidelity/hifi
# /scripts for launch

This PR makes some changes to scripting to improve the new user experience and help scripters get started quickly.  

Everything that was previously in the  ```/examples``` folder  remains available in ```/archivedScripts```
 
A ```Reveal Scripts Folder``` button in the "Running Scripts" dialog will open the ```/scripts``` folder, which currently has four subfolders https://github.com/highfidelity/hifi/pull/7728

- ```/default``` -  the current default scripts, with all references and assets stored locally!
- ```/developer```  - diagnostics, tools, utilities, etc.  References assets that are generally on individual developer hosting (s3)
- ```/tutorials``` - simple scripts that anyone can hack on Day 1.  Most create an entity and then stop themselves.  Some avatar control scripts run until they are stopped. Entity scripts are stored on production s3 bucket so that others users can see results
- ```/templates``` - collect our best patterns &practices and abstract them into easy to use templates

To test:
1. Stop all scripts
2. Run "defaultScripts.js" from /scripts/default
3. Make sure all default functionality works as expected.
4. Try loading each of the /tutorials
5. Test some developer tools
